rbc/src/main/java/com/library/rbc/controller/BookController.java

```diff
@@ -48,4 +52,10 @@ public Page<BookDto> returnBooksBy(@ParameterObject @PageableDefault(size = 12)
       @RequestParam(required = false) List<String> bookStatuses) {
     return bookService.getBooksBy(pageable, bookCategories, bookStatuses);
   }
+
+  @PostMapping("/upload-image")
+  public ResponseEntity<ImageDto> uploadImage(@RequestPart MultipartFile image) {
+    String result = bookService.uploadImage(image);
+    return ResponseEntity.status(HttpStatus.OK).body(new ImageDto(result));
+  }
```

Comment:
This can be simplified to:
```java
public ImageDto uploadImage(@RequestPart MultipartFile image) {
    String result = bookService.uploadImage(image);
    return new ImageDto(result);
  }
```